python/tests/integration/arcticdb/version_store/test_symbol_list.py

```diff
@@ -48,9 +48,9 @@ def small_max_delta():
         unset_config_int("SymbolList.MaxDelta")
 
 
-def make_read_only(lib):
-    return NativeVersionStore.create_store_from_lib_config(
-        lib.lib_cfg(), Defaults.ENV, OpenMode.READ
+def make_read_only(lib, native_config):
+    return NativeVersionStore.create_store_from_lib_and_static_config(
```

Comment:
Breaking change to create_store_from_lib_config?

Only used in our code in Man though. Similar to other comments, I think the clean way to do this is to extend the types allowed in the first argument rather than allowing a new argument.